source/html/renderer/src/host-config.ts

```diff
@@ -133,101 +166,123 @@ function parseFactTitleDefinition(obj: any): IFactTitleDefinition {
     return result;
 }
 
-export interface IFactSetConfig {
-    title: IFactTitleDefinition,
-    value: IFactTextDefinition,
-    spacing: number
+export class FactSetConfig {
+    title: FactTitleDefinition = new FactTitleDefinition();
+    value: FactTextDefinition = new FactTextDefinition();
+    spacing: number;
 }
 
-function parseFactSetConfiguration(obj: any): IFactSetConfig {
+function parseFactSetConfiguration(obj: any): FactSetConfig {
```

Comment:
Since FactSetConfig is now a class, this method should be moved to the class and should be renamed into simply "parse"